registry/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.9
+
+COPY ./ /usr/src
+
+WORKDIR /usr/src
+RUN pip install -r requirements.txt
+
+# Start web server
+CMD [ "uvicorn","main:app","--host", "0.0.0.0", "--port", "80" ]
```

Comment:
I remember RBAC service and API service should share same docker image, @windoze is this correct?

Reply:
@windoze will help to package the docker image for ui, access control plugin, multi-option registry.

Reply:
Then let's remove this file.